assets/js/ajax.js

```diff
@@ -1,18 +1,20 @@
-jQuery(document).ready(function ($) {
-    if( $('body').hasClass('archive')){
-        return;
-    }
-    $.ajax({
-        url: wp_post_views_ajax_object.ajaxurl, // this is the object instantiated in wp_localize_script function
-        type: 'POST',
-        data: {
-            action: 'wppv_counter', // this is the function in your functions.php that will be triggered
-            post_id: wp_post_views_ajax_object.post_id,
-            nonce: wp_post_views_ajax_object.nonce,
-        },
-        success: function (data) {
+document.addEventListener('DOMContentLoaded', function () {
+    if (false == document.body.classList.contains("archive")) {
+        let url = wp_post_views_ajax_object.ajaxurl; // this is the object instantiated in wp_localize_script function);
+
+        let data = new FormData();
+        data.append('action', 'wppv_counter');
+        data.append('post_id', p_post_views_ajax_object.post_id);
```

Comment:
Please use `wp_post_views_ajax_object.post_id`

Reply:
This commit fixed that: https://github.com/vanpariyar/wp-post-views/pull/33/commits/7249d20e1473390694b67d98c3830a3e08a40389